### PR TITLE
GCE: update test config for the Windows job

### PR DESCRIPTION
**Pull request by yujuhong**
- Exclude [LinuxOnly] or [Serial] tests
- Include [sig-windows] tests
- Update NUM_NODES (i.e., number of Linux nodes)

### CODE REVIEW

File: config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml [hunk 1]
```diff
@@ -74,13 +74,15 @@ periodics:
...
       - --test-cmd=$GOPATH/src/k8s.io/gce-k8s-windows-testing/run-e2e.sh
-      - --test-cmd-args=--ginkgo.focus=\[Conformance\] --minStartupPods=8
+      - --test-cmd-args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\] --ginkgo.skip=\[LinuxOnly\]|\[Serial\] --minStartupPods=8
       - --timeout=120m
       env:
       - name: KUBE_GCE_ENABLE_IP_ALIASES
         value: "true"
       - name: NUM_WINDOWS_NODES
         value: "3"
+      - name: NUM_NODES
```
Comment (dashpole) [inline config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: just curious; why do we still run linux nodes in a windows cluster?

Comment (yujuhong) [reply config/jobs/kubernetes/sig-gcp/sig-gcp-windows.yaml]: To run all our wonderful cluster addons that don't run on Windows yet

**Pull request merged**